storm-core/src/clj/org/apache/storm/daemon/acker.clj

```diff
@@ -18,74 +18,24 @@
   (:import [org.apache.storm.tuple Tuple Fields])
```

Comment:
In clojure testing, creation of acker bolt will be overridden in with-tracked-cluster. After porting the implementation of acker to java, it seems there is no way to do a similar override for java method. So, the mk-acker-bolt of acker.clj was kept as temporary solution to make the clojure testing pass. It can be removed after the porting of common.clj and relative testing modules. Is there a solution or workaround for this issue?